### ISSUE TITLE
Mineflayer detected that you are using a deprecated event (physicTick)! Please use this event (physicsTick) instead.

### ISSUE THREAD

**Issue by rekosah** [kolbytn/mindcraft#167]
How can i even use this event 

Mineflayer detected that you are using a deprecated event (physicTick)! Please use this event (physicsTick) instead.

**Comment by CatManXVZ** [comment]
I have the same issue

**Comment by jiveshkalra** [comment]
Any Fix? Same here

**Comment by jiveshkalra** [comment]
Yup same here, the problem is not with mineflayer , its just a warning , 
the actual problem might be that you are using invalid characters in your bot name like " "(space) is also invalid
n that was the problem for me

**Comment by BF5258** [comment]
> Any Fix? Same here

The problem is in Mindcraft's dependencies rather than Mindcraft itself.
Change `physicTick` to `physicsTick` in the following locations:
- mindcraft\node_modules\mineflayer-pvp\lib\PVP.js line 51
- node_modules\mineflayer-collectblock\lib\CollectBlock.js line 95
- There might be others I havn't noticed.

If you want to create a .patch file you can then use the following commands:
`npx patch-package mineflayer-pvp`
`npx patch-package mineflayer-collectblock`

**Comment by uukelele** [comment]
Fixed in #219
